fabtests/scripts/runfabtests.sh

```diff
@@ -556,6 +556,10 @@ function set_cfg_file {
 		name=$1
 	fi
 
+	if [[ $parent == 'sockets' ]]; then
```

Comment:
@shefty Thoughts on this omission? Would you prefer automatically switching to quick for sockets or to reduce the all.test configs for sockets?

Reply:
I think this change is a hack.  :)  There should instead be a config file for sockets that works, not a silent switch to some other test file.

Reply:
I don't disagree but to be clear this is not "silently switching." This is only hit when the user does not explicitly set the config file so it defaults to quick because all.test is too long because sockets. all.test *should* work but exposes a bug and takes years. I'm still not opposed to reducing the tests in all.test but just pointing it out.

Reply:
There is a  `-u` option to specify the config file for `fi_ubertest`.

Reply:
@wzamazon Thank you, I totally missed that.